cmd/query/app/flags.go

```diff
@@ -36,18 +36,22 @@ import (
 )
 
 const (
+	queryHostPort           = "query.host-port"
 	queryPort               = "query.port"
 	queryBasePath           = "query.base-path"
 	queryStaticFiles        = "query.static-files"
 	queryUIConfig           = "query.ui-config"
 	queryTokenPropagation   = "query.bearer-token-propagation"
 	queryAdditionalHeaders  = "query.additional-headers"
 	queryMaxClockSkewAdjust = "query.max-clock-skew-adjustment"
+	queryPortWarning        = "(deprecated, will be removed after 2020-06-30 or in release v1.20.0, whichever is later)"
```

Comment:
nit: I would place it next to queryPort const

Reply:
per #2113, we want to set deprecation to now+3m

Reply:
I will change that. So deprecated at 2020-08-31?

Reply:
yes